rim/forms.py

```diff
@@ -0,0 +1,15 @@
+from django import forms
+
+from rim.models import Equipment
+
+class AddForm(forms.ModelForm):
+
+    class Meta:
+        model = Equipment
+        fields = ['serial_no', 'equipment_model', 'equipment_type', 'count', 'manufacturer',
```

Comment:
You should change this to `exclude = []` to make things easier when we add/delete any fields on the Equipment model